Musical Collection/Musical Collection Console App/Interfaces/Classes Interfaces/IAlbum.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Musical_Collection_Console_App.Classes;
+using Musical_Collection_Console_App.Interfaces.Classes_Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Musical_Collection_Console_App.Interfaces
+{
+    public interface IAlbum : IEntity
```

Comment:
Please privede documentation on all non private Classes, Methods and Properties :)